examples/case-management.v1.test.js

```diff
@@ -0,0 +1,422 @@
+/**
+* @jest-environment node
+*/
+/**
+ * (C) Copyright IBM Corp. 2021.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const CaseManagementV1 = require('../dist/case-management/v1');
+const { readExternalSources } = require('ibm-cloud-sdk-core');
+const authHelper = require('../test/resources/auth-helper.js');
+
+// Location of our config file.
```

Comment:
Don't forget to put the comments re: the external config properties here.

Reply:
Added in https://github.com/IBM/platform-services-node-sdk/pull/77/commits/61230f6c97e95f14d1e34db1aaaf443db2c33b3b